### PR TITLE
Erlang exceptions fix

### LINKED ISSUE #13: UndefinedFunctionError: undefined function: :badarg.__struct__/0 (module :badarg is not available)
We're having an error when parsing logs using the logfmt library:

https://github.com/jclem/logfmt-elixir/issues/6

When those errors happen, we also see the following in our logs, and the error is not reported (using the `use_logger` option):

```
01:02:03.208 [warn]  Unable to notify Honeybadger! UndefinedFunctionError: undefined function: :badarg.__struct__/0 (module :badarg is not available)
```

### PR DESCRIPTION
**Pull request by joshuap**
This fixes #13.

### CODE REVIEW

**Comment by joshuap** [comment]
[Exception.normalize](https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/exception.ex#L85) to the *rescue*. :)

File: lib/honeybadger/notice.ex [hunk 1]
```diff
@@ -4,6 +4,7 @@ defmodule Honeybadger.Notice do
   defstruct [:notifier, :server, :error, :request]
 
   def new(exception, metadata \\ %{}, backtrace) do
+    exception = Exception.normalize(:error, exception)
```
Comment (rbishop) [inline lib/honeybadger/notice.ex]: Nice find! I did not know about that function.

**Comment by rbishop** [comment]
Looks good to me :+1:

**Comment by joshuap** [comment]
Thanks!

**Pull request merged**